src/python/pants/backend/python/goals/coverage_py.py

```diff
@@ -313,6 +313,23 @@ def get_branch_value_from_config(fc: FileContent) -> bool:
     return cp.getboolean(run_section, "branch", fallback=False)
 
 
+def get_namespace_value_from_config(fc: FileContent) -> bool:
```

Comment:
Can you add a unit tests for this to `coverage_py_test.py`? There's already a `test_get_branch_value_from_config` that you could take inspiration from

Reply:
Sorry, forgot that!